src/_guides/language/effective-dart/documentation.md

```diff
@@ -277,21 +277,32 @@ makes an API easier to learn.
 ### DO use square brackets in doc comments to refer to in-scope identifiers.
 
 If you surround things like variable, method, or type names in square brackets,
-then dartdoc will look up the name and link to its docs.
+then dartdoc looks up the name and links to the entity's docs:
 
 {:.good-style}
 <?code-excerpt "misc/lib/effective_dart/docs_good.dart (identifiers)"?>
 {% prettify none %}
 /// Throws a [StateError] if ...
-/// similar to [anotherMethod], but ...
+/// similar to [anotherMethod()], but ...
 {% endprettify %}
 
-You can also link to constructors using `new` followed by the constructor:
+You can put parentheses after the name too, which makes it clearer when you're
+referring to a method or constructor. You can link to a specific member of a
+class using the class name and member name, separated by a dot:
 
 {:.good-style}
-<?code-excerpt "misc/lib/effective_dart/docs_good.dart (new)"?>
+<?code-excerpt "misc/lib/effective_dart/docs_good.dart (member)"?>
 {% prettify none %}
-/// To create a point, call [new Point] or use [new Point.polar] to ...
+/// Similar to [Duration.inDays], but handles fractional days.
+{% endprettify %}
+
+Named constructors work like members. For the unnamed constructor, put
```

Comment:
The first sentence seems too imprecise. How about:

Refer to named constructors the same way you refer to methods. To refer to the unnamed constructor, ...